src/node_file.cc

```diff
@@ -454,9 +454,8 @@ int FileHandle::ReadStart() {
 
     // Push the read wrap back to the freelist, or let it be destroyed
     // once we’re exiting the current scope.
-    constexpr size_t wanted_freelist_fill = 100;
     auto& freelist = handle->binding_data_->file_handle_read_wrap_freelist;
-    if (freelist.size() < wanted_freelist_fill) {
+    if (freelist.size() < /* wanted_freelist_fill */ 100) {
```

Comment:
For this, using the constant would be most consistent with our common style, but the constant could be renamed... e.g.

```c++
constexpr size_t kWantedFreelistFill = 100;
//...
if (freelise.size() < kWantedFreelistFill) {
```